MAUI/Calendar/accessibility.md

```diff
@@ -0,0 +1,162 @@
+---
+layout: post
+title: Accessibility in .NET MAUI Calendar control | Syncfusion
+description: Learn here all about accessibility features of Syncfusion .NET MAUI Calendar (SfCalendar) control. 
```

Comment:
description: Learn here all about the accessibility features of Syncfusion .NET MAUI Calendar (SfCalendar) control.